frontend/src/components/InfoPanel.jsx

```diff
@@ -161,8 +163,8 @@ class InfoPanel extends React.PureComponent {
       `Set counterfactual lockdown date to ${this.state.dateLockdownCounterfactual}`
     );
     await Promise.all([
-      this.loadStatisticsCounterfactual(),
       this.loadAllowedDates(),
+      this.loadStatisticsCounterfactual(),
```

Comment:
As above, I think we might need to drop `Promise.all` if one of these needs to finish before the other one starts.

Reply:
This is now implemented in 9436143.